yarn.lock

```diff
@@ -1237,15 +1238,15 @@
   resolved "https://registry.yarnpkg.com/@expo/config-types/-/config-types-43.0.1.tgz#3e047dccb371741a540980eaff26fb0c95039c30"
   integrity sha512-EtllpCGDdB/UdwAIs5YXJwBLpbFQNdlLLrxIvoILA9cXrpQMWkeDCT9lQPJzFRMFcLUaMuGvkzX2tR4tx5EQFQ==
 
-"@expo/config@6.0.6", "@expo/config@^6.0.6":
-  version "6.0.6"
-  resolved "https://registry.yarnpkg.com/@expo/config/-/config-6.0.6.tgz#64b49b93f07cb046f5a8538a1793bef9070d8d52"
-  integrity sha512-GPI8EIdMAtZ5VaB4p5GcfuX50xyfGFdpEqLi0QmcfrCfTsGry1/j/Qy28hovHM1oJYHlaZylTcbGy+1ET+AO2w==
+"@expo/config@6.0.14", "@expo/config@^6.0.14", "@expo/config@^6.0.6":
```

Comment:
What's still referencing `@expo/config@^6.0.6`? (yarn why)

Reply:
No idea:
```
expo 𝝠 yarn why @expo/config        
yarn why v1.22.17
[1/4] 🤔  Why do we have the module "@expo/config"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "@expo/config@6.0.14"
info Reasons this module exists
   - "_project_#expo-updates" depends on it
   - Hoisted from "_project_#expo-updates#@expo#config"
   - Hoisted from "_project_#jest-expo-puppeteer#@expo#config"
   - Hoisted from "_project_#jest-expo#@expo#config"
   - Hoisted from "_project_#expo-constants#@expo#config"
   - Hoisted from "_project_#expo-yarn-workspaces#@expo#metro-config#@expo#config"
   - Hoisted from "_project_#expo-splash-screen#@expo#prebuild-config#@expo#config"
   - Hoisted from "_project_#expo-yarn-workspaces#@expo#webpack-config#@expo#config"
   - Hoisted from "_project_#jest-expo-puppeteer#xdl#@expo#config"
   - Hoisted from "_project_#expo-yarn-workspaces#@expo#webpack-config#expo-pwa#@expo#config"
info Disk size without dependencies: "72.49MB"
info Disk size with unique dependencies: "77.06MB"
info Disk size with transitive dependencies: "82.45MB"
info Number of shared dependencies: 60
✨  Done in 1.82s.
```

Reply:
There are few Expo packages which requires exactly this version of config (without any semver ranges) at this time, for example:
* `@expo/metro-config` - https://github.com/expo/expo/blob/master/yarn.lock#L1335-L1344
* `@expo/prebuild-config` - https://github.com/expo/expo/blob/master/yarn.lock#L1392-L1405
* `@expo/webpack-config` - https://github.com/expo/expo/blob/master/yarn.lock#L1476-L1509
* `expo-pwa` - https://github.com/expo/expo/blob/master/yarn.lock#L8040-L8049